base_layer/core/src/blocks/blockheader.rs

```diff
@@ -89,12 +89,10 @@ impl BlockHeader {
     pub fn validate_pow(&self) -> bool {
         unimplemented!();
     }
-}
 
-impl Default for BlockHeader {
-    fn default() -> Self {
+    pub fn new(blockchain_version: u16) -> BlockHeader {
```

Comment:
If we pass something in here should we rather not pass in the whole concensus struct? 
Perhaps we need to change stuff later on as well, like the pow